plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/DefaultFreeExtensions.php

```diff
@@ -824,47 +824,72 @@ public static function get_plugin( $slug ) {
-				'learn_more_link' => 'https://woocommerce.com/products/mailpoet',
+				'label'            => __( 'Reach your customers with MailPoet', 'woocommerce' ),
+				'image_url'        => plugins_url( '/assets/images/core-profiler/logo-mailpoet.svg', WC_PLUGIN_FILE ),
+				'description'      => __( 'Send purchase follow-up emails, newsletters, and promotional campaigns.', 'woocommerce' ),
+				'learn_more_link'  => 'https://woocommerce.com/products/mailpoet',
+				'install_priority' => 7,
 			),
 			'tiktok-for-business'           => array(
-				'label'           => __( 'Create ad campaigns with TikTok', 'woocommerce' ),
-				'description'     => __( 'Create advertising campaigns and reach one billion global users.', 'woocommerce' ),
-				'learn_more_link' => 'https://woocommerce.com/products/tiktok-for-woocommerce',
+				'label'            => __( 'Create ad campaigns with TikTok', 'woocommerce' ),
+				'image_url'        => plugins_url( '/assets/images/core-profiler/logo-tiktok.svg', WC_PLUGIN_FILE ),
+				'description'      => __( 'Create advertising campaigns and reach one billion global users.', 'woocommerce' ),
+				'learn_more_link'  => 'https://woocommerce.com/products/tiktok-for-woocommerce',
+				'install_priority' => 1,
 			),
 			'google-listings-and-ads'       => array(
-				'label'           => __( 'Drive sales with Google Listings & Ads', 'woocommerce' ),
-				'description'     => __( 'Reach millions of active shoppers across Google with free product listings and ads.', 'woocommerce' ),
-				'learn_more_link' => 'https://woocommerce.com/products/google-listings-and-ads',
+				'label'            => __( 'Drive sales with Google Listings & Ads', 'woocommerce' ),
+				'image_url'        => plugins_url( '/assets/images/core-profiler/logo-google.svg', WC_PLUGIN_FILE ),
+				'description'      => __( 'Reach millions of active shoppers across Google with free product listings and ads.', 'woocommerce' ),
+				'learn_more_link'  => 'https://woocommerce.com/products/google-listings-and-ads',
+				'install_priority' => 6,
 			),
 			'woocommerce-services:tax'      => array(
-				'label'           => __( 'Get automated tax rates with WooCommerce Tax', 'woocommerce' ),
-				'description'     => __( 'Automatically calculate how much sales tax should be collected – by city, country, or state.', 'woocommerce' ),
-				'learn_more_link' => 'https://woocommerce.com/products/tax',
+				'label'            => __( 'Get automated tax rates with WooCommerce Tax', 'woocommerce' ),
+				'image_url'        => plugins_url( '/assets/images/core-profiler/logo-woo.svg', WC_PLUGIN_FILE ),
+				'description'      => __( 'Automatically calculate how much sales tax should be collected – by city, country, or state.', 'woocommerce' ),
+				'learn_more_link'  => 'https://woocommerce.com/products/tax',
+				'install_priority' => 4,
 			),
 		);
 
+		// Copy shipping for the core-profiler and remove is_visible conditions, except fo the country restriction.
```

Comment:
```suggestion
		// Copy shipping for the core-profiler and remove is_visible conditions, except for the country restriction.
```

Reply:
Hey, @moon0326. Do you have the context for why we need to copy shipping and remove `is_visible`?

Wouldn't it be better if we apply the change in the `get_plugin` function instead?

Reply:
@ilyasfoo I think we should do that in the future when we have completely removed the profile wizard. I would like to preserve the existing code as much as possible in case we encounter an issue and need to switch back to the profile wizard.